test/powershell/Language/Classes/Scripting.Classes.RunPath.Tests.ps1

```diff
@@ -0,0 +1,42 @@
+Describe "Script with a class definition run path" -Tags "CI" {
+
+    It "Script with a class definition can run from a path without a comma" {
+
+        $FilePath = '.\MyTest.ps1'
+
+        try {
+            'class MyClass { [string]$MyProperty }; $True' | Out-File -FilePath $FilePath
+            $Success = . $FilePath
+        }
+
+        catch {
+            $Success = $False
+        }
+
+        finally {
+            Remove-Item -Path $FilePath
+        }
+
+        $Success | Should Be $True
+    }
+
+    It "Script with a class definition can run from a path with a comma" {
```

Comment:
These two test cases can be combined into 1 using the -TestCases parameter for It. Example: https://github.com/PowerShell/PowerShell/blob/02b5f357a20e6dee9f8e60e3adb9025be3c94490/test/powershell/Language/Parser/LanguageAndParser.TestFollowup.Tests.ps1#L147